tools/deploy/common.sh

```diff
@@ -25,8 +28,10 @@ register_certificate() {
 
   if [ "$properties" != '' ]; then
     while [ "$status" != '0' ] && [ $retry -lt 3 ]; do
-      $REGISTER_CERTIFICATE_BIN --config $properties
-      status="$?"
+      status_code=$($REGISTER_CERTIFICATE_BIN --config $properties | jq -r '.status_code')
+      if [ ${status_code} = "OK" ] || [ ${status_code} = "CERTIFICATE_ALREADY_REGISTERED" ]; then
```

Comment:
In this tool handles set of commands are independent groups so it will try to register the same certificate repeatedly.
So we will consider the already registered response as a success.
Example:-
[command/register_company.json](https://github.com/scalar-labs/scalar-ist/tree/main/tools/deploy_tool/src/main/resources/command/register_company.json) and [command/upsert_user_profile_admin.json](https://github.com/scalar-labs/scalar-ist/tree/main/tools/deploy_tool/src/main/resources/command/upsert_user_profile_admin.json) are trying to register same certificate.